### PR TITLE
How do I get into heaven?

### LINKED ISSUE #18: Explain "by grace you have been saved through faith"
Paul writes ([Eph 2:8 ESV][1]), "For by grace you have been saved through faith." Does this mean Christians who proclaim Jesus to be their savior are guaranteed to be saved?

[1]: https://www.esv.org/Ephesians+2/

### PR DESCRIPTION
**Pull request by martyychang**
This branch was initially created to resolve #18 but it turns out the conversation is naturally tied to #22 because salvation and heaven are concepts difficult to separate.

### CODE REVIEW

File: manuscript/SAVED.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+Q: How do I get into heaven?
+A: Jesus saves the faithful. So just be faithful to God.
+
+## Commentary ##
+
+We're all going to die one day. Morbid, perhaps, or just real. Death means our physical bodies are going to fail, our hearts will stop, our neurons will stop firing, and life as we know it will come to an end.
+
+But God promises an afterlife. To be clear, God tells is that your soul will live on, heading toward a final judgment at which point you will be either granted eternal life (entry into heaven) or condemned to eternal death (no heaven for you). The only problem is that standard of conduct required for entry is high, unattainable high. So high, in fact, that you and I have already f***ed up.
```
Comment (DavidLozzi) [inline manuscript/SAVED.md]: Typo - *God tells is that

Comment (DavidLozzi) [reply manuscript/SAVED.md]: Suggestion to not curse or even hint at it. James 3:9, keep this focused on his glory and praise. Suggested replacements: messed up, fowled up, dropped the vall, etc. ;)

**Review by DavidLozzi — COMMENTED** [review]
Great start. Check out Matthew‬ ‭7:21-23‬, how does this answer how we get in? Why do some think they might get in? Faith yes but James 2:14+. More than faith

File: manuscript/SAVED.md (hunk 1 above)
Comment (martyychang) [reply manuscript/SAVED.md]: Fixed the typo

Comment (martyychang) [reply manuscript/SAVED.md]: Removed the bad word

**Comment by martyychang** [comment]
Hi @DavidLozzi would you take another look? I've fixed the two inline comments you made, and I've also incorporated Mt 7:21-23 and Jas 2:14-17.

**Comment by martyychang** [comment]
While this content warrants further refinement, in the agile spirit the pr will be merged with a future iteration to come.

**Pull request merged**